src/app/components/builder/templates/one-page-standard/logic/contents/education.content.logic.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
   UNDEFINED_PERIOD_END
 } from '../../constants/one-page-standard.constants';
 import { SAMPLE_RESUME } from '../../constants/sample-resume.constants.spec';
-import { startCase, upperCase } from 'lodash';
+import { startCase, toUpper } from 'lodash';
```

Comment:
what's the difference between this two?

Reply:
upperCase make it space separated, whereas toUpper just change from lower to upper case while preserving punctuations. If we type "Works Applications Co., Ltd", upperCase will make it "WORKS APPLICATIONS CO LTD" while toUpper will make it "WORKS APPLICATIONS CO., LTD.".